### ISSUE TITLE
Unable to add bot to server

### ISSUE THREAD

**Issue by galacticfrequency** [toBeOfUse/SpellingBeeBot#1]
![Screenshot_20240809-075029](https://github.com/user-attachments/assets/451dec01-9da3-442e-a92e-df568b24742f)

**Comment by toBeOfUse** [comment]
Yes. I have been afraid to ask Discord to verify this bot because I worry I will get in trouble for letting people pirate a New York Times puzzle and the bot will just be deleted. But I will try, for you

**Comment by galacticfrequency** [comment]
Totally, I understand. It's no problem if it isn't able to be verified.
It's better that it works on few servers than none at all.

Le lun. 12 août 2024, 12 h 57 a.m., Mitch J. ***@***.***> a
écrit :
